examples/python/ColumnOperations/README.md

```diff
@@ -205,13 +208,59 @@ The second function is an aggregation and sums the values in a column. We iterat
         # Yield the row data constructed
         yield SSE.BundledRows(rows=[SSE.Row(duals=duals)])
 ```
+### `MaxOfColumns_2` function
+The `MaxOfColumns_2` function computes the maximum in each of two columns and returns the maximum values in two columns, therefore making it appropriate to be used from the Qlik load script. As you can see the function also set the TableDescription header before sending the result.
```

Comment:
As you can see, the function also sets the TableDescription header before sending the result.